tests/Api/RenderTest.php

```diff
@@ -69,7 +69,7 @@ public function testRenderFromQuery_TemplateKey_DataKey() {
   public function testRenderFromQuery_TemplateKey_DataId() {
     $fixtures = $this->loadFixtures();
     $templateKey = 'T';
-    $dataId = $fixtures['docData'][0]['dataId'];
+    $dataId = $fixtures['docData'][0]['id'];
     $uri = self::ROUTE."?docType={$this->docType}&templateKey={$templateKey}&dataId={$dataId}";
```

Comment:
should the url parameters be key?

Reply:
Since we're not dealing directly with a template, kind of like the compiled templates, I like to have more descriptive parameters here. if I saw "key" and "id" here, I wouldn't know that one is for a template and the other is for data.